docs/reference/commandline/exec.md

```diff
@@ -14,6 +14,7 @@ Run a command in a running container
 Options:
   -d, --detach         Detached mode: run command in the background
       --detach-keys    Override the key sequence for detaching a container
+  -e, --env=[]         Set environment variables
```

Comment:
Can you update the man-page as well? https://github.com/docker/docker/blob/master/man/docker-exec.1.md